generate/authorize.js

```diff
@@ -0,0 +1,834 @@
+/* 
+ * Authorization Code Generator functions
+ * 
+ * @module create-graphql-server-authorization
+ * @export function generateAuthCode(typeName, inputSchema)
+ *
+ */
+
+// name of the @authorize directive, which triggers authorization logic
+ const AUTHORIZE_DIRECTIVE = 'authorize';
+
+// constants to read GraphQL Abstract Syntax Tree (AST)
+ const OBJECT_TYPE_DEFINITION = 'ObjectTypeDefinition';
+ const FIELD_DEFINITION = 'FieldDefinition';
+ const NON_NULL_TYPE = 'NonNullType';
+ const STRING_VALUE = 'StringValue';
+ const LIST_TYPE = 'ListType';
+ const LIST_VALUE = 'ListValue';
+ const DIRECTIVE = 'Directive';
+ const ARGUMENT = 'Argument';
+ const NAME = 'Name';
```

Comment:
I'm pretty sure you can get these constants out of the `graphql` package.

Reply:
yes, I do now.